source/developers/cook-books/graphql/index.rst

```diff
@@ -0,0 +1,202 @@
+.. _working_with_graphql:
+
+====================
+Working with GraphQL
+====================
+
+Crafter CMS provides built-in support for GraphQL to query content in any site without writing additional code.
+A GraphQL schema is generated independently for each site based on the content-type configuration that has been 
+created using Crafter Studio, and the schema is automatically updated after any change is detected.
+
+To implement a site that uses GraphQL you would follow a workflow like this:
+
+1. Create a new site (if needed, for existing sites skip to step number 3)
+2. Define the content model for your site
+3. Obtain the GraphQL schema for your site, you can use the provided GraphiQL client or any third party client
+4. Develop GraphQL queries to use in your site or external app
+
+All content changes made by authors in Crafter Studio will be immediately available in GraphQL queries.
+
+When a change is made in the content model, for example adding a new field or creating a new content-type, the
+GraphQL schema will be rebuilt to reflect the same changes. So for a Crafter CMS site that uses GraphQL queries the
+develop process would look like this:
+
+1. Developers make changes in the content model
+2. Developers update the GraphQL queries to use the latest schema
+3. Authors make updates using the latest content model
+4. Publishers review & approve the author's work
+5. Publishers publish to live both the content model configuration & the content updates
+6. Crafter Deployer will handle the GraphQL schema rebuild in delivery
+
+You can also use the Crafter CMS GraphQL API from an external site or application, however in this case you will need to
```

Comment:
What's this about?

Reply:
Steps 5 & 6 are about publishing the form-definition.xml files to rebuild the schema in delivery.

The last paragraph is about handling when using the API outside of a crafter site